stdlib/public/core/Dictionary.swift

```diff
@@ -1897,7 +1897,7 @@ extension Dictionary.Index: Comparable {
 }
 
 extension Dictionary.Index: Hashable {
-  @inlinable
+  @_effects(readonly) // FIXME(cocoa-index): Make inlinable
```

Comment:
Is this correct? It writes to the hasher, after all.

Reply:
Oh, oops, good catch!